### DISCUSSION TITLE
Rancher desktop on VMware VDI

### DISCUSSION THREAD

**Discussion by ViktorMikhalkin** [rancher-sandbox/rancher-desktop#3070] [General]
Hi Team,

Want to try Rancher to substitute Docket Desktop. The question is if it's possible to run Rancher Desktop against Windows 10 being run on VMware VDI.

Many thanks in advance.

**Comment by jandubois** [comment]
I don't know VMware VDI, but I think it should be possible as long as there is a way to enable "Nested Virtualization" for it.

It is possible to run Rancher Desktop this way in VMware Fusion and VMware Workstation.
